lib/ts/recipe/emailpassword/emailPassword.ts

```diff
@@ -158,26 +158,23 @@ export default class EmailPassword extends RecipeModule {
      */
 
     async signUpValidate(input: APIFormField[]): Promise<FormFieldError[]> {
-        return await validateFormOrThrow(input, this.config.signInAndUpFeature.signUpForm.formFields);
+        return await validateForm(input, this.config.signInAndUpFeature.signUpForm.formFields);
```

Comment:
This can throw an error in case the input fields do not have the same length as the config form-field. In this case, what will happen?

Ideally, you should stick to only throwing an error in a function from a function or not throwing one at all. But OK.

Reply:
The app would crash in that case. That should only happen while developers are developing a custom theme, trying to submit an incomplete form to the backend during development.
This is fairly important that the custom theme is implemented correctly and that this error is not being ignored so I was thinking that it's fine to throw unexpectedly here. Let me know if that's ok for you